librclone/librclone.go

```diff
@@ -79,5 +82,11 @@ func RcloneRPC(method *C.char, input *C.char) (result C.struct_RcloneRPCResult)
 	return result
 }
 
+// Use this to free the string returned by RcloneRPC
+//export FreeString
+func FreeString(str *C.char) {
```

Comment:
Can we call this `RcloneFreeString` - I've been namespacing the C symbols starting with `Rclone`.